modules/build/src/test/scala/scala/build/tests/BuildTests.scala

```diff
@@ -572,29 +605,10 @@ class BuildTests extends munit.FunSuite {
     }
   }
 
-  test("ScalaNativeOptions for native-gc with no values") {
```

Comment:
Removed test?

Reply:
This is not valid syntax of using directives and it's gonna break in parser. Idk if we should test such case there or in my library

Reply:
shouldn't we make key optional that default to True?